### ISSUE TITLE
dirname doesn't work on windows

### ISSUE THREAD

**Issue by viperguynaz** [googlesamples/google-photos#21]
On windows with Node v16.13.1, in app.js, line 35:
`const dirname = path.dirname(new URL(import.meta.url).pathname);`
yields: '/C:/your-project...' and all associated mappings 404.

I changed line 35 to:
`const dirname = path.resolve();`
which correctly yields: 'C:/your-project...' - drops leading '/'
This works on windows.

**Comment by jfschmakeit** [comment]
Thanks for filing this issue. 

I'm not sure if `path.resolve()` is robust enough here as it works off the current working directory, but not the absolute path of the file. However, if you are running the app from the directory the `app.js` file is located in, then this should work. 

We will do some testing on Windows and update the sample accordingly.

**Comment by ugur1yildiz** [comment]
Another method may be as follows;
```js
import { fileURLToPath } from 'url';
import { dirname } from 'path';

const __filename = fileURLToPath(import.meta.url);
const dirname1 = dirname(__filename);
```

**Comment by jfschmakeit** [comment]
@ugur1yildiz Yes, that's similar to what we have settled on. Thanks for sharing your suggestion.

Some of the team is still out of the office at the moment, but I'm hoping to get this fix merged as soon as possible. For now, using @viperguynaz 's workaround should work if you are starting the app from the `REST/PhotoFrame` directory directly.

**Comment by jfschmakeit** [comment]
This has been fixed in commit 6b6ffcb using `fileURLToPath`. Please try again and let us know if you are still having issues running this sample on Windows. 

Thanks!
